packages/template-retail-react-app/app/components/list-menu/index.test.js

```diff
@@ -5,66 +5,53 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import React from 'react'
-import {screen, fireEvent} from '@testing-library/react'
+import {screen, waitFor} from '@testing-library/react'
 import ListMenu from './index'
-import {renderWithProviders} from '../../utils/test-utils'
+import {renderWithProviders, setupMockServer} from '../../utils/test-utils'
+
+// jest.mock('../../contexts/index', () => {
+//     const originalModule = jest.requireActual('../../contexts/index')
+//     return {
+//         ...originalModule,
+//         fetchCategoryNode: jest.fn().mockReturnValue({
+//             id: 'mens-clothing',
+//             name: 'Clothing',
+//             loaded: true
+//         })
+//     }
+// })
+
+jest.mock('../../commerce-api/utils', () => {
+    const originalModule = jest.requireActual('../../commerce-api/utils')
+    return {
+        ...originalModule,
+        isTokenValid: jest.fn().mockReturnValue(true)
+    }
+})
 
-const mockRoot = {
-    id: 't1',
-    name: 'Test One',
-    categories: [
-        {
-            id: 't1-1',
-            name: 'Level one Test One One'
-        },
-        {
-            id: 't1-2',
-            name: 'Level one Test One Two',
-            categories: [
-                {
-                    id: 't1-2-1',
-                    name: 'Level two Test One Two One',
-                    categories: [
-                        {
-                            id: 't1-2-1-1',
-                            name: 'Level three Test One Two One One'
-                        },
-                        {
-                            id: 't1-2-2-2',
-                            name: 'Level three Test One Two Two Two'
-                        }
-                    ]
-                },
-                {
-                    id: 't1-2-2',
-                    name: 'Level two Test One Two Two'
-                }
-            ]
-        }
-    ]
-}
+const server = setupMockServer()
 
 describe('ListMenu', () => {
-    test('ListMenu renders without errors', () => {
-        renderWithProviders(<ListMenu root={mockRoot} />)
+    test('ListMenu renders without errors', async () => {
+        renderWithProviders(<ListMenu />)
 
-        const popoverTrigger = document.querySelector('[id^="popover-trigger"]')
-        fireEvent.mouseOver(popoverTrigger)
+        const drawer = document.getElementById('chakra-toast-portal')
 
+        const category = await waitFor(() => screen.getByText(/Clothing/i))
```

Comment:
What is the full page expected to look like? Given the nature of the content, the test might be more reliable if we search for something more specific than the word "clothing".